src/Components/ArticleDetail.jsx

```diff
@@ -0,0 +1,41 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { getIndividualArticle } from "../Utils/api"
+import ErrorPage from "./ErrorPage";
+
+export default function ArticleDetail() {
+
+    const [isLoading, setIsLoading] = useState(true);
+    const [selectedArticle, setSelectedArticle] = useState(null);
+    const [error, setError] = useState(null)
+
+    const { article_id } = useParams()
+
+    useEffect(() => {
+        getIndividualArticle(article_id)
+        .then((article) => {
+            setSelectedArticle(article)
+            setIsLoading(false)
+        })
+        .catch((err) => {
```

Comment:
should loading still be true if there is an error? It probably wont matter for this case but if you were to refactor you could end up with the wrong things rendering